src/components/Trade/TradeConfirm/TradeConfirm.tsx

```diff
@@ -146,13 +146,13 @@ export const TradeConfirm = ({ history }: RouterProps) => {
   const txLink = useMemo(() => {
     switch (trade.sources[0].name) {
       case 'Osmosis':
-        return `${osmosisAsset?.explorerTxLink}${buyTxid}`
+        return `${osmosisAsset?.explorerTxLink}${sellTxid}`
       case 'CowSwap':
         return `https://explorer.cow.fi/orders/${sellTxid}`
       default:
         return `${trade.sellAsset?.explorerTxLink}${sellTxid}`
     }
-  }, [trade, sellTxid, osmosisAsset, buyTxid])
+  }, [trade, sellTxid, osmosisAsset])
```

Comment:
Revert and make it a separate PR?

Reply:
Why? That just slows things down

Reply:
Are you really asking to split up a PR that is adding and removing 4 lines of code 😂

Reply:
https://github.com/shapeshift/web/pull/2576